examples/graticule.js

```diff
@@ -22,8 +23,21 @@ const map = new Map({
   })
 });
 
+const swipe = document.getElementById('swipe');
```

Comment:
I think it is better to keep the example simple instead of adding this feature to it. Better document the option with a sensible example snippet.